metadata-analysis/Metadata-Analysis.Rmd

```diff
@@ -139,17 +168,114 @@ counts_by_week %>%
   geom_point() +
   geom_text(nudge_x = .5, color = "gray15", check_overlap = TRUE) +
   scale_color_brewer(palette = "Dark2") +
-  scale_x_date(date_breaks = "1 week", date_labels = "%m-%d") +
+  scale_x_date(date_breaks = "1 week", date_labels = "%m/%d") +
   labs(
-    title = "Eva Site Hits",
+    title = "Weekly Eva Site Hits",
     x = "Week Ending",
     y = ""
   ) +
   theme_minimal() +
+  theme(legend.position = "bottom") +
+  ylim(0, NA)
+  
+```
+
+```{r fig.width=10, fig.height=5}
+
+counts_by_month %>%
+  filter(Event == "Site hits") %>%
+  ggplot(aes(x = MonthEnding, y = n, group = Event, color = Event, label = n)) +
+  geom_line() +
+  geom_point() +
+  geom_text(nudge_x = .5, color = "gray15", check_overlap = TRUE) +
+  scale_color_brewer(palette = "Dark2") +
+  scale_x_date(
+    date_breaks = "1 month",
+    date_labels = "%B") +
+  labs(
+    title = "Monthly Eva Site Hits",
+    x = "Month Ending",
+    y = ""
+  ) +
+  theme_minimal() +
   theme(legend.position = "bottom")
   
 ```
 
+## Tabs Visited
+
+```{r fig.width=7, fig.height=6}
+
+staging_last6months %>%
+  filter(str_detect(Event, "User on") & 
+           !(Event %in% c("User on tabHome", "User on tabUpload", "User on"))
+         & !str_detect(Event, "DEMO MODE")) %>%
+  distinct(SessionID, Event, .keep_all = TRUE) %>%
+  mutate(Tab = case_when(
+    str_detect(Event, "tabChangelog") ~ "Changelog",
+    str_detect(Event, "tabClientCount") ~ "Client Counts",
+    str_detect(Event, "tabDQOrg") ~ "Organization DQ",
+    str_detect(Event, "tabDQSystem") ~ "System DQ",
+    str_detect(Event, "tabLocalSettings") ~ "Local Settings",
+    str_detect(Event, "tabPDDE") ~ "PDDE DQ",
+    TRUE ~ as.character(Event)), 
+    Tab = factor(Tab, levels = c("Local Settings", "Client Counts", "PDDE DQ", "System DQ", "Organization DQ", "Changelog"))) %>%
+  count(Tab) %>%
+  ggplot(aes(x = Tab, y = n, fill = Tab)) +
+  geom_bar(stat = "identity") +
+  geom_text(aes(label = scales::percent(n/sum(n))), color = "gray15", vjust = -0.5) +
+  scale_fill_manual(values = 
+                   c("Changelog" = "#f94144",
+                      "Client Counts" = "#f3722c",
+                      "Organization DQ" = "#f8961e",
+                      "System DQ" = "#f9c74f",
+                      "Local Settings" = "#90be6d",
+                      "PDDE DQ" = "#43aa8b")) +
+ labs(title = "Tabs Visited in the Last 6 Months",
+       x = "Tab",
+       y = "Count") +
+  theme_minimal() +
+  theme(legend.position = "bottom") +
+  guides(fill = guide_legend(ncol = 2))
+
+```
+
+```{r fig.width=7, fig.height=6}
+
+staging %>%
```

Comment:
this plot doesn't work for me. the filter leaves me with 0 rows

Reply:
fixed